ndcube/utils/wcs.py

```diff
@@ -10,7 +10,40 @@
 from astropy import wcs
 from astropy.wcs._wcs import InconsistentAxisTypesError
 
-__all__ = ['WCS', 'reindex_wcs', 'add_celestial_axis']
+__all__ = ['WCS', 'reindex_wcs', 'add_celestial_axis', 'wcs_ivoa_mapping']
+
+
+class TwoWayDict(dict):
```

Comment:
I suggest you replace this with:
```python
from collections import UserDict


class TwoWayDict(UserDict):
    @property
    def inv(self):
        """"
        The inverse dictionary.
        """"
        return {v:k for k, v in self.items()}
```

As it's API compatible with https://github.com/jab/bidict and rather cleaner than this one.

Reply:
OK.  Done.